docs/metadata/git/git-resolve.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Resolve conflicts
+id: git-resolve
+description: Source Control
+sidebar_position: 4
+tags:
+  - metadata
+  - Git
+  - resolve
+---
+
+This page describes how to resolve conflicts that you may run into while merging your changes.
+
+## Resolve conflicts during merging
+
+Now let's create a Project in `third_branch` with same name as `first_Pipeline` and put a different path in Dataset.
+Post this once we pull the changes from `main` into this branch, it would have conflicts as `first_Pipeline` exists both in
+`main` and `current` branch.
+
+One way is to use the Git interface directly to resolve conflicts between branches.
+
+Or we can use the simple merge UI on Prophecy to resolve the conflicts
+
+Let's see in the video below on how to resolve conflicts using simple merge on Prophecy:
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://user-images.Githubusercontent.com/103921419/174733075-b878a77d-97df-4bc9-8baf-5fcc72e268db.mp4" title="How to resolve conflicts" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+
+### Resolve conflicts using manual merge
+
+For SQL, you can use manual merge to resolve conflicts. This provides you with a simple, yet effective way to resolve merge conflicts for granular changes.
+
+![Choose a Git conflict manual merge](img/git-conflict-manual-merge.png)
+
+- **(A)** **Strategy** - You must choose a preferred strategy to resolve the conflict. Here the Left strategy keeps the version on branch `master`, while the Right strategy keeps the version on branch `dev`.
```

Comment:
The left/right strategy is available for resolving conflicts in spark as well. The difference is manual merge by making code changes directly is available only in sql

Reply:
Done.